scale/job/configuration/interface/configuration_interface.py

```diff
@@ -0,0 +1,96 @@
+"""Defines the interface for defining a default job configuration"""
+from __future__ import unicode_literals
+
+import logging
+
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+from job.configuration.interface.exceptions import InvalidInterfaceDefinition
+
+from error.models import Error
+
+logger = logging.getLogger(__name__)
+
+CONFIGURATION_INTERFACE_SCHEMA = {
+    'type': 'object',
+    'additionalProperties': False,
+    'properties': {
+        'version': {
+            'description': 'version of the configuration_interface schema',
+            'type': 'string',
+            'pattern': '^.{0,50}$',
+        },
+        'default_settings': {
+            'type': 'object',
+            'item': {
+                'type': 'string',
+            },
+        },
+    },
+}
+
+
+class ConfigurationInterface(object):
+    """Represents the interface for defining a default job configuration"""
```

Comment:
I would rename this to JobTypeConfiguration (and change all applicable code with the configuration interface name).